src/components/Card.jsx

```diff
@@ -1,11 +1,54 @@
+import React from 'react';
+import IconButton from '@mui/material/IconButton';
+import Popover from '@mui/material/Popover';
+import InfoOutlined from '@mui/icons-material/InfoOutlined';
+import Box from '@mui/material/Box';
+
 import './Card.scss';
 
-const Card = ({ children, header }) => {
+const Card = ({ children, header, helptip }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const isOpen = Boolean(anchorEl);
+
   return (
     <div className="Card">
       {header && (
         <div className="Header">
-          <h6>{header}</h6>
+          <h6>
+            {header}
+            {helptip && (
```

Comment:
so `helptip` is only observed if `header` is? worth some jsdoc on `Card`

Reply:
done